packages/styletron-engine-snapshot/src/index.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+
+export {default as Client} from "./client/client.js";
```

Comment:
Given that the client and server implementation are currently identical, I think this could just export the same module under both `Client` and `Server` aliases.

Reply:
done